universal-login-contracts/lib/message.ts

```diff
@@ -17,11 +17,12 @@ export const messageToUnsignedMessage = (message: Partial<Message>): UnsignedMes
     nonce: message.nonce!,
     gasPrice: message.gasPrice!,
     gasToken: message.gasToken!,
-    gasBase: 0,
-    gasCall: 0,
+    gasLimit: message.gasLimit!,
```

Comment:
As I understand we can update messageToUnsignedMessage signature to `(message:Message): UnsignedMessage`,
because this message used only in `calculateGasBase` where type of arg is `Omit<Message, 'gasLimit'>` and all fields in variable `message` are required. So we also don't need gasLimit in messageWithoutGasEstimates

Reply:
We need `gasLimit` to calculate `gasCall`. See line 23.